tmux/record_ros_bag.yml

```diff
@@ -0,0 +1,61 @@
+#
+# Used to create ROS bags for all topics published from the HL2.
```

Comment:
We may want to comment that this is to be run by itself, e.g. not in combination with another tmuxinator launch. Maybe in the future we change this or create a new version that may be run in combination, but lets not venture that far yet and just document what we have.